analyzer/tests/functional/analyze_and_parse/test_analyze_and_parse.py

```diff
@@ -187,3 +189,55 @@ def check_one_file(self, path, mode):
             print("Failed to run: " + ' '.join(cerr.cmd))
             print(cerr.output)
             return cerr.returncode
+
+    def test_json_output_for_macros(self):
```

Comment:
Does the `--print-steps` flag change the returned json somehow? Is the bug path always in the json output?

Reply:
No, it will not change it. The bugpath and all the data will be always in the json output.

What do you think? Do we have to remove these field from the output and only print them if the --print-steps option is given?

Reply:
No, I think we can keep it as it is now.